front/app/global-styles.ts

```diff
@@ -6,6 +6,12 @@ const GlobalStyle = createGlobalStyle`
     box-sizing: border-box;
   }
 
+  .admin-user-view {
+    .weglot_switcher {
+      margin-right: 60px;
+    }
+  }
+
```

Comment:
This was the only solution I could think of, since we don't have a weglot component or anything in our codebase :thinking: If you have a different suggestion though, I'm definitely open to something else!